### PR TITLE
README: cleanup

### PR DESCRIPTION
**Pull request by vladdoster**
Various grammatical fixes and edits to increase readability.

- correct comma usage
- remove verbiage (e.g.,_simply_ and _just_) to create stronger intent
- correct word order

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,7 +141,7 @@ To install only a few of the available utilities:
 $ make UTILS='UTILITY_1 UTILITY_2' install
 ```
 
-To install every program with a prefix (e.g. uu-echo uu-cat):
+To install every program with a prefix (e.g., uu-echo uu-cat):
```
Comment (Arcterus) [inline README.md:144]: Is this actually required, or is it just stylistic?

File: README.md [hunk 2]
```diff
@@ -83,8 +83,7 @@ you want to build manually.  For example:
 $ cargo build --features "base32 cat echo rm" --no-default-features
 ```
 
-If you don't even want to build the multicall binary and would prefer to just
-build the utilities as individual binaries, that is possible too.  For example:
+If you don't even want to build the multicall binary and prefer to build the utilities as individual binaries, that is possible.  For example:
```
Comment (Arcterus) [inline README.md:86]: I’d prefer if you add the “too” back, but I’m fine with this otherwise.

File: README.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ aggregate GNU coreutils rewrites.
 Why?
 ----
 
-Many GNU, Linux and other utilities are useful, and obviously
+Many GNU, Linux, and other utilities are useful and obviously
```
Comment (Arcterus) [inline README.md:24]: Aren’t these both independent clauses?

File: README.md (hunk 1 above)
Comment (vladdoster) [reply README.md:144]: Here is a thread on it 

https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e

File: README.md (hunk 3 above)
Comment (sylvestre) [reply README.md:24]: @vladdoster ping?

**Comment by sylvestre** [comment]
Closing as no activity 
feel free to reopen

**Pull request closed without merging**